appinventor/appengine/src/com/google/appinventor/client/editor/youngandroid/actions/RefreshCompanionAction.java

```diff
@@ -6,13 +6,35 @@
 package com.google.appinventor.client.editor.youngandroid.actions;
 
 import com.google.appinventor.client.Ode;
+import com.google.gwt.dom.client.NativeEvent;
+import com.google.gwt.event.dom.client.KeyCodes;
 import com.google.gwt.user.client.Command;
+import com.google.gwt.user.client.Event;
 
 public class RefreshCompanionAction implements Command {
+
+  private RefreshCompanionAction() {
```

Comment:
Previously, this constructor was public. What is the purpose of making it private now?

Reply:
I changed it to private because this class is not being instantiated by any other code.